source/php/Component/Slider__item/slider__item.json

```diff
@@ -17,7 +17,8 @@
         "focusPoint": {"top": 30, "left": 50},
         "textAlignment": "",
         "overlay": "none",
-        "showPauseButton": true
+        "showPauseButton": true,
+        "isImage": false
```

Comment:
Is this really neccesary? We do have image parameter. May we use that to auto-detect this behaviour?